google/cloud/storage/client.h

```diff
@@ -1937,11 +1931,12 @@ class Client {
    * https://cloud.google.com/storage/docs/access-control/create-manage-lists#defaultobjects
    */
   template <typename... Options>
-  void DeleteDefaultObjectAcl(std::string const& bucket_name,
-                              std::string const& entity, Options&&... options) {
+  StatusOr<void> DeleteDefaultObjectAcl(std::string const& bucket_name,
+                                        std::string const& entity,
+                                        Options&&... options) {
     internal::DeleteDefaultObjectAclRequest request(bucket_name, entity);
     request.set_multiple_options(std::forward<Options>(options)...);
-    raw_client_->DeleteDefaultObjectAcl(request).value();
+    return raw_client_->DeleteDefaultObjectAcl(request).status();
```

Comment:
This one is a little funny as it highlights the (surprising?) difference between StatusOr<void>::StatusOr(Status) and StatusOr<T>::StatusOr(Status).  We should find out how absl::StatusOr<void> is going to behave.

Reply:
Ack.